src/platform/SystemEventSupport.cpp

```diff
@@ -29,12 +29,11 @@
 
 namespace chip {
 namespace System {
-namespace Platform {
```

Comment:
Why remove event handling functions from following namespace?
namespace Platform {
namespace Eventing {

But put CHIP_ERROR StartTimer(System::Layer & aLayer, uint32_t aMilliseconds) into this namespace?

Reply:
This is still needed for now because of the Platform→Inet→System dependency layering.